### PR TITLE
Refactor project bug report template

### PR DESCRIPTION
**Pull request by DeveloperWOW64**

### CODE REVIEW

**Review by mousemaid — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -61,12 +61,10 @@ body:
       label: 📟 Stack Trace
       description: Copy and paste the stack trace from FluentHub's error page.
       value: |
-        "details><summary>Stack trace details</summary><div>
-        
+        <details><summary>Stack trace details</summary><div>
         ```
-        paste here
+        <!--paste here-->
```
Comment (onein528) [inline .github/ISSUE_TEMPLATE/bug_report.yml:66]: In code block, it doesn't make any sence.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -61,12 +61,10 @@ body:
       label: 📟 Stack Trace
       description: Copy and paste the stack trace from FluentHub's error page.
       value: |
-        "details><summary>Stack trace details</summary><div>
-
```
Comment (onein528) [inline .github/ISSUE_TEMPLATE/bug_report.yml:65]: Why did you delete double line breaks and double-quatations?

**Review by onein528 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (DeveloperWOW64) [reply .github/ISSUE_TEMPLATE/bug_report.yml:65]: why shouldn't I have deleted quotation mark?

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (DeveloperWOW64) [reply .github/ISSUE_TEMPLATE/bug_report.yml:66]: ohhh I forgot that..

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (onein528) [reply .github/ISSUE_TEMPLATE/bug_report.yml:65]: Ah, Already '|' is there. Never mind.

> How about 'Why did you delete double line breaks?'

Comment (DeveloperWOW64) [reply .github/ISSUE_TEMPLATE/bug_report.yml:65]: > How about 'Why did you delete double line breaks?'

Why was it necessary in the first place?

Comment (onein528) [reply .github/ISSUE_TEMPLATE/bug_report.yml:65]: It doesnt seem to be recongnized as a code block.

Comment (onein528) [reply .github/ISSUE_TEMPLATE/bug_report.yml:65]: Here

image


image
